src/generator/AutoRest.Ruby/TemplateModels/MethodTemplateModel.cs

```diff
@@ -334,6 +334,18 @@ public IEnumerable<ParameterTemplateModel> LocalParameters
         }
 
         /// <summary>
+        /// Get the parameters that are actually method parameters in the order they appear in the method signature
+        /// exclude global parameters and constants
+        /// </summary>
+        public IEnumerable<ParameterTemplateModel> MethodParameters
```

Comment:
Good catch. 
--Removed `LocalParameters` as it's a dead variable. Kept the name `MethodParamters` for clarity. 
--Also removed the comments as now we are supporting param grouping.